### PR TITLE
[13.0][FIX] mail_activity_team: activity count

### PR DESCRIPTION
**Pull request by chienandalu**
The user couldn't see the proper count of his own activities. It was
allways showing the team activities counter.

cc @Tecnativa TT35885

please review @Tardo @victoralmau  @MiquelRForgeFlow @LoisRForgeFlow 

Also please forward port features across versions so you don't miss them later, like this one https://github.com/OCA/social/pull/877

### CODE REVIEW

File: mail_activity_team/models/res_users.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class ResUsers(models.Model):
 
     @api.model
     def systray_get_activities(self):
-        if not self._context.get("team_activities", False):
+        if not self.env.context.get("team_activities"):
```
Comment (MiquelRForgeFlow) [inline mail_activity_team/models/res_users.py:17]: why this change? 🤔

Comment (chienandalu) [reply mail_activity_team/models/res_users.py:17]: It's been several months from the patch :sweat_smile: but probably there was an issue with cached context

Comment (pedrobaeza) [reply mail_activity_team/models/res_users.py:17]: You should not use the internal variable, but the external one. It has the same content for now, but that may change in the future.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@MiquelRForgeFlow can you please approve?

**Review by MiquelRForgeFlow — APPROVED** [review]
